src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -2369,6 +2376,129 @@ def find_a(i):
     return williamson_goethals_seidel_skew_hadamard_matrix(a, b, c, d, check=check)
 
 
+def skew_hadamard_matrix_spence_1975(n, existence=False, check=True):
+    r"""
+    Construct a skew Hadamard matrix of order `n = 4(1 + q + q^2)` using Spence construction.
```

Comment:
```suggestion
    Construct a skew Hadamard matrix of order `n = 4(1 + q + q^2)` using the Spence construction.
```
Possibly also needs to be split somewhere to drop to 80 char/line.